app/src/androidTest/java/com/github/campus_capture/bootcamp/fragments/MapColorTest.java

```diff
@@ -0,0 +1,139 @@
+package com.github.campus_capture.bootcamp.fragments;
+
+import static androidx.test.espresso.Espresso.onView;
+import static com.github.campus_capture.bootcamp.authentication.Section.AR;
+import static com.github.campus_capture.bootcamp.authentication.Section.CGC;
+import static com.github.campus_capture.bootcamp.authentication.Section.EL;
+import static com.github.campus_capture.bootcamp.authentication.Section.GC;
+import static com.github.campus_capture.bootcamp.authentication.Section.GM;
+import static com.github.campus_capture.bootcamp.authentication.Section.IN;
+import static com.github.campus_capture.bootcamp.authentication.Section.MA;
+import static com.github.campus_capture.bootcamp.authentication.Section.MT;
+import static com.github.campus_capture.bootcamp.authentication.Section.MX;
+import static com.github.campus_capture.bootcamp.authentication.Section.NONE;
+import static com.github.campus_capture.bootcamp.authentication.Section.PH;
+import static com.github.campus_capture.bootcamp.authentication.Section.SC;
+import static com.github.campus_capture.bootcamp.authentication.Section.SIE;
+import static com.github.campus_capture.bootcamp.authentication.Section.SV;
+
+import android.content.Context;
+import android.content.Intent;
+
+import androidx.test.espresso.action.ViewActions;
+import androidx.test.espresso.intent.Intents;
+import androidx.test.espresso.matcher.ViewMatchers;
+import androidx.test.ext.junit.rules.ActivityScenarioRule;
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.GrantPermissionRule;
+
+import com.github.campus_capture.bootcamp.R;
+import com.github.campus_capture.bootcamp.activities.MainActivity;
+import com.github.campus_capture.bootcamp.authentication.Section;
+import com.github.campus_capture.bootcamp.firebase.BackendInterface;
+import com.github.campus_capture.bootcamp.scoreboard.ScoreItem;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class MapColorTest {
+
+    private final BackendInterface mockColor = new BackendInterface() {
```

Comment:
Same here use the PlaceholderBackend and you can remove all the override methods doing nothing and returning null